src/main/resources/application.properties

```
@@ -1,4 +1,7 @@
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.url=jdbc:postgresql://localhost:5432/TelegramBot_db
+spring.datasource.username=student
+spring.datasource.password=chocolatefrog
 telegram.bot.token=
+spring.liquibase.change-log=classpath:liquibase/changelog-master.yml
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
при использовании liquibase стратегия должна быть validate, иначе смысла от liquibase нет никакого - его можно просто удалить из проекта